readthedocs/core/management/commands/update_repos.py

```diff
@@ -56,7 +58,7 @@ def handle(self, *args, **options):
         force = options['force']
         version = options['version']
         if args:
```

Comment:
Here we need to check `if options` now instead of `if args` otherwise it fails in a weird way :)